doc/release/upcoming_changes/24858.new_feature.rst

```diff
@@ -0,0 +1 @@
+Extend ``np.add`` to work with ``unicode`` and ``bytes`` dtypes.
```

Comment:
```suggestion
* Extend ``np.add`` to work with ``unicode`` and ``bytes`` dtypes.
```
Otherwise, it should render oddly (even if we edit the notes anyway at some point likely)